tests/lib/tools/nested-state

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: prepare"
+    echo "       restore"
+    echo "       build-image IMAGE-TYPE"
+    echo "       boot-image IMAGE-TYPE [--param-cd PARAM] [--param-mem PARAM]"
+    echo "       start-vm"
+    echo "       shutdown-vm"
+    echo "       delete-vm"
+    echo ""
+    echo "Available options:"
+    echo "  -h --help   show this help message."
+    echo ""
+    echo "COMMANDS:"
```

Comment:
Thank you for adding the help message. This is much more clear now.

Reply:
yaw